bcp/client/src/webpack/helper/map-display.js

```diff
@@ -258,3 +270,160 @@ export function getAircraftSilhouettePath(aircraft) {
+    // .status-planned-dispatch
+    // .status-standby
+    // .status-available
+    // .status-unserviceable
+
+    // These are the eventTypes.
+    // UNKOWNN
+    // AVAILABLE
+    // UNAVAILABLE
+    // LIMITED
+    // DEPLOYED
+    // PLANNED_DISPATCH
+    // STANDBY
+    // STANDBY_AMENDED
+    // STANDBY_TEMP_ASSET
+    // UNSERVICEABLE
+    // UNSERVICEABLE_AIRDESK
+    // UNSERVICEABLE_STOOD_DOWN
+    // UNSERVICEABLE_RESTRICTED
+    // UNSERVICEABLE_STANDBY
+
+    switch (eventType) {
+        case 'AVAILABLE':
+        case 'LIMITED':
+            return 'available';
+        case 'DEPLOYED':
+            return 'dispatched';
+        case 'PLANNED_DISPATCH':
+            return 'planned-dispatch';
+        case 'STANDBY':
+        case 'STANDBY_AMENDED':
+        case 'STANDBY_TEMP_ASSET':
+            return 'standby';
+        case 'UNSERVICEABLE':
+        case 'UNSERVICEABLE_AIRDESK':
+        case 'UNSERVICEABLE_STOOD_DOWN':
+        case 'UNSERVICEABLE_RESTRICTED':
+        case 'UNSERVICEABLE_STANDBY':
+            return 'unserviceable';
+        default:
+            return 'unknown';
+    }
+}
+
+export function buildPopUpAircraftCommonContent(aircraft, includeTitle) {
+    const aircraftData = {includeTitle: includeTitle};
+    if (includeTitle) {
+        aircraftData.title = getAssetTitle(aircraft.properties);
+    }
+    aircraftData.lastSeen = getAssetLastSeen(aircraft);
+    aircraftData.spatialDisp = getAssetSpatialDisp(aircraft);
+    aircraftData.details = getAircraftDetails(aircraft.properties);
+    return require('./templates/assetCommonContent.hbs')(aircraftData);
+}
+
+export function buildPopUpEquipmentCommonContent(equipment, includeTitle) {
+    const equipmentData = {includeTitle: includeTitle};
+    if (includeTitle) {
+        equipmentData.title = getAssetTitle(equipment);
+    }
+    equipmentData.lastSeen = getAssetLastSeen(equipment);
+    equipmentData.spatialDisp = getAssetSpatialDisp(equipment);
+    equipmentData.details = getEquipmentDetails(equipment);
+    equipmentData.contactDetails = getAssetDispatchContactDetails(equipment);
+    return require('./templates/assetCommonContent.hbs')(equipmentData);
+}
+
+export function getFuellingArrangmentDisplayValue(fuellingArrangement) {
+    if (!fuellingArrangement) {
+        return '';
+    }
+    switch (fuellingArrangement) {
+        case 'D':
+            return 'Dry Only';
+
+        case 'BD':
+            return 'Wet-B or Dry';
+
+        case 'ABD':
+            return 'Wet-A, Wet-B or Dry';
+
+        case 'NA':
+            return 'Not Applicable';
+
+        default:
+            return '';
+    }
+}
+
+export function getAssetAvailabilityNotes(asset) {
+    if (asset && asset.properties.notes) {
+        // Remove double and training newlines.
+        let fullNotes = asset.properties.notes.replace(/\n\n/g, '\n').trim();
+
+        if (fullNotes !== '') {
+            let truncatedNotes = truncateNotes(fullNotes, 60);
```

Comment:
I think you could probably just use css instead of all this string manipulation.
`white-space: pre-line;` instead of replacing \n with <br/>
`text-overflow: ellipsis;` instead of truncating